rules/detekt/src/test/kotlin/com/twitter/compose/rules/detekt/ComposeModifierWithoutDefaultCheckTest.kt

```diff
@@ -27,7 +27,7 @@ class ComposeModifierWithoutDefaultCheckTest {
         val errors = rule.lint(composableCode)
         assertThat(errors).hasSourceLocations(
             SourceLocation(2, 15),
-            SourceLocation(4, 57)
+            SourceLocation(4, 46)
```

Comment:
dunno why this was 57 before?? just looked at the code it points, and 46 points at the `modifier2` name which is what we wanted?